### PR TITLE
New package: gscope3

### PR DESCRIPTION
**Pull request by cr6git**

### CODE REVIEW

**Comment by cr6git** [comment]
grumble.. same error as [#7451](https://github.com/voidlinux/void-packages/pull/7451).

**Comment by cr6git** [comment]
@Johnnynator provided the [theoretical solution](https://github.com/voidlinux/void-packages/pull/7451#issuecomment-323548800) to get rid of this error, but I don't know how to make use of it, I never wrote any patch...

**Comment by Johnnynator** [comment]
Creating a patch isn't that hard:
First find the file that has to be patched (in this case packages/gscope/src/auto_gen.c) and make  a backup. Then just edit the original file as needed (adding the 3 lines).
The last step is running `diff -u backupfile editedfile` and put the output into the patches folder, you can also redirect the output directly into a file.
```
-- auto_gen.c  2017-08-20 17:26:08.340009975 +0200
+++ packages/gscope/src/auto_gen.c      2017-08-20 17:27:13.104011572 +0200
@@ -25,6 +25,11 @@
 #include "display.h"
 #include "app_config.h"

+//musl doesn't define ACCESPERMS
+#ifndef ACCESSPERMS
+# define ACCESSPERMS (S_IRWXU|S_IRWXG|S_IRWXO) 
+#endif
+
 //===============================================================
 //       Defines
 //===============================================================
```

**Comment by cr6git** [comment]
I learn something new every day!

File: srcpkgs/gscope3/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+build_style=gnu-configure
+hostmakedepends="automake pkg-config"
+makedepends="gtksourceview-devel"
+maintainer="cr6git <quark6@protonmail.com>"
+short_desc="Interactive source code browsing tool for C and other languages"
+homepage="https://github.com/tefletch/gscope"
+license="MIT"
+distfiles="https://github.com/tefletch/gscope/archive/${_release}.tar.gz"
+checksum=e530438a42998c3f4f61468f8d917377a22861f39c9b338e42c021a5efe925ed
+wrksrc="gscope-2.10-v3.10"
```
Comment (the-maldridge) [inline srcpkgs/gscope3/template]: for consistency please put  wrksrc and build_wrksrc lines below _release

Comment (the-maldridge) [reply srcpkgs/gscope3/template]: nvm, I just fixed it in the merge.

**Pull request closed without merging**